### ISSUE TITLE
Customize height of mode and header lines?

### ISSUE THREAD

**Issue by noctuid** [alexluigit/dirvish#45]
I'd like to make the mode line a little taller.

I also use a header line for buffers instead of a mode line, and the height of my header line shown in the preview window is a little smaller than the height of dirvish's header line.  I'd like to be able to control the height to make them match.

There is `dirvish-header-line-text-size`, but it does not seem to work anymore?  Also, more fine-grained control would be nice if possible (like `doom-modeline-height`).

**Comment by alexluigit** [comment]
> There is dirvish-header-line-text-size, but it does not seem to work anymore? Also, more fine-grained control would be nice if possible (like doom-modeline-height).

I've wanted to achieve this for very long time.  Because of the window minimal height issue mentioned in #41, I tried different hacks to handle the header/footer, but none of them were satisfactory. Finally I found a way to solve this. I've removed `dirvish-header-line-text-size`, this hack is no longer needed. Now we can adjust the mode/header line height by the new options `dirvish-mode-line-height`, `dirvish-header-line-height`. For users who want to have different text-height, they can customize the relevant face themselves.

We now also have `dirvish-header-line-position`, same as `dirvish-mode-line-position`, but for header line.
